src/bin/vip-cache-purge-url.js

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env node
+
+/**
+ * @flow
+ * @format
+ */
+
+/**
+ * External dependencies
+ */
+
+/**
+ * Internal dependencies
+ */
+import { appQuery, purgeCache } from 'lib/api/cache-purge';
+import command from 'lib/cli/command';
+import { trackEvent } from 'lib/tracker';
+import { readFromFile } from 'lib/read-file';
+import * as exit from 'lib/cli/exit';
+
+const examples = [
+	{
+		usage: 'vip cache purge-url <URL>',
+		description: 'Purge cache for a URL',
```

Comment:
Today's session taught me that we should be very specific with our terms :)

```suggestion
		description: 'Purge a URL from page cache',
```

Reply:
@chriszarate Much better on all suggestions, thanks! :)